mirage/factories/curriculum-inventory-sequence.js

```diff
@@ -0,0 +1,5 @@
+import { Factory } from 'ember-cli-mirage';
+
+export default Factory.extend({
+  description: (i) => `descirption for sequence ${i} `,
```

Comment:
typo: should be "description"

Reply:
Existing issue. I will fix post-merge unless there is something else that is going to require running tests again anyway.